.github/workflows/ci-plugin-server.yml

```diff
@@ -134,4 +134,88 @@ jobs:
                   # Below DB name has `test_` prepended, as that's how Django (ran above) creates the test DB
                   DATABASE_URL: 'postgres://posthog:posthog@localhost:5432/test_posthog'
                   REDIS_URL: 'redis://localhost'
-              run: cd plugin-server && yarn test --shard=${{matrix.shard}}
+              run: cd plugin-server && yarn test --runInBand --forceExit tests/ --shard=${{matrix.shard}}
```

Comment:
nit: these CI changes would have made nicely for a separate PR to keep things small

Reply:
I ended up doing this to avoid contamination from the other tests, which don't use auto ids in places and hence end up having id collisions. It all got a bit out of hand, sorry!